battery_management_systems/lg_resu_prime_http/README.md

```diff
@@ -0,0 +1,11 @@
+# LG RESU (10|16)h Prime (http)
+
+This [Enapter Device Blueprint](https://go.enapter.com/marketplace-readme) integrates **LG RESU 10h and 16h Prime** lithium battery control and monitoring via an undocumented http interface on the battery
+
+Use an Enapter Virtual UCM to load this blueprint and set the IP address of the LG Battery.
```

Comment:
@muenchris Can I add **Connect to Enapter** section to your README file?

Reply:
yes, I also recommend being a bit more relaxed on the formatting if you want adoption of your marketplace since most tools are not auto-formating the way you want it. This is very much blocking productivity and will result in much longer times until your library will fill with 3rd party blueprints.